### ISSUE TITLE
Make landing page/docs more actionable

### ISSUE THREAD

**Issue by ryanclark2** [disclose/disclose#31]
I don't have specific suggestions, but as a <user?> coming to this page/project, I think it could be more clear what I can do to get started. Thoughts?

**Comment by dantrauner** [comment]
Definitely interested in suggestions – we can probably do a better job in terms of a more prominent CTA. The last panel on the page tries to achieve this, but it's probably not obvious enough visually, and it doesn't provide much insight into what types of contributions we're looking for.

<img width="566" alt="screen shot 2019-02-20 at 9 18 56 am" src="https://user-images.githubusercontent.com/1462423/53110908-e036e480-34f0-11e9-98df-15e93f0fabaa.png">

We're actually looking at rendering the [list of programs](https://github.com/disclose/disclose/blob/master/bug-bounty-list/bug-bounty-list.json) as a more visually-appealing list on the landing page itself (specifically to highlight the safe harbor status of each program). Once we do that, I think we'll want to make it especially clear how to add/remove/change an entry.

**Comment by ryanclark2** [comment]
I saw that and wonder if most folks looking at this would understand what it means to open an issue on GH. Honestly not sure who would be looking so don't know. Not sure if having an email option would be a lower barrier to entry? Or an email form? Probably would invite a lot of spam. 

Definitely into the list of programs. I looked through a few on the list and none of them had the logo. Not sure if I'm looking in the right spot. Cool to see those policies though. Assuming that's kind of the intention behind this project.
